packages/next/src/lib/metadata/generate/meta.tsx

```diff
@@ -55,8 +55,12 @@ function camelToSnake(camelCaseStr: string) {
 
 function getMetaKey(prefix: string, key: string) {
   // Use `twitter:image` and `og:image` instead of `twitter:image:url` and `og:image:url`
-  // to be more compatible as it's a more common format
-  if ((prefix === 'og:image' || prefix === 'twitter:image') && key === 'url') {
+  // to be more compatible as it's a more common format.
+  // `og:video` & `og:audio` do not have a `:url` suffix alias
+  if (
+    ['og:image', 'twitter:image', 'og:video', 'og:audio'].includes(prefix) &&
```

Comment:
Can we make it a const set and define it outside of function, and use has here. It will save some perf

```
const aliasPropPrefixes = new Set([...])
function getMetaKey(prefix: string, key: string) {
  // ...
  if (aliasPropPrefixes.has(prefix) && ...)
}
```